tests/plotting/test_plotting.py

```diff
@@ -123,6 +124,28 @@ def verify_cache_image(plotter):
                       f'{error}')
 
 
+@skip_not_vtk9
+@skip_no_plotting
+def test_pbr(sphere):
+    """Test PBR rendering"""
+    texture = examples.load_globe_texture()
+
+    pl = pyvista.Plotter(lighting=None)
+    pl.set_environment_texture(texture)
+    pl.add_light(pyvista.Light())
+    pl.add_mesh(sphere,
+                color='w',
+                pbr=True, metallic=0.8, roughness=0.2,
+                smooth_shading=True,
+                diffuse=1)
+    pl.add_mesh(pyvista.Sphere(center=(0, 0, 1)),
+                color='w',
+                pbr=True, metallic=0.0, roughness=1.0,
+                smooth_shading=True,
+                diffuse=1)
+    pl.show(before_close_callback=verify_cache_image)
```

Comment:
This looks awesome! Might be a waste to have only in a test :)

Reply:
Yes, it does look cool.  I'll consider moving it over.

Reply:
Not moving it over since the globe texture isn't a cubemap.